flink-python/src/main/java/org/apache/flink/python/AbstractPythonFunctionRunner.java

```diff
@@ -224,4 +235,15 @@ public JobBundleFactory createJobBundleFactory(Struct pipelineOptions) throws Ex
 	public abstract ExecutableStage createExecutableStage() throws Exception;
 
 	public abstract OutputReceiverFactory createOutputReceiverFactory();
+
+	/**
+	 * Gets the proto representation of the base MetricGroup used for all user-defined functions.
```

Comment:
What about remove the `used for all user-defined functions` as metrics will be used not only for UDF?